pkg/autoscaler/statforwarder/forwarder.go

```diff
@@ -293,40 +281,23 @@ func (f *Forwarder) setProcessor(bkt string, p *bucketProcessor) {
 func (f *Forwarder) createProcessor(bkt, holder string) *bucketProcessor {
 	if holder == f.selfIP {
 		return &bucketProcessor{
+			bkt:    bkt,
+			logger: f.logger,
```

Comment:
should we annotate logger with some key here?
Same below.

Reply:
Added a bucket key.